site2/docs/admin-api-brokers.md

```diff
@@ -4,16 +4,26 @@ title: Managing Brokers
 sidebar_label: Brokers
 ---
 
+> **Important**
+>
+> This page only shows **some frequently used operations**.
+>
+> - For the latest and complete information about `pulsar-admin`, including commands, flags, descriptions, and more, see [here](https://pulsar.apache.org/tools/pulsar-admin/).
+> 
+> - For the latest and complete information about `REST API`, including parameters, responses, samples, and more, see [here]{@inject: rest:REST:/}.
+> 
+> - For the latest and complete information about `Java admin API`, including classes, methods, descriptions, and more, see [here](https://pulsar.apache.org/api/admin/).
+
 Pulsar brokers consist of two components:
 
 1. An HTTP server exposing a {@inject: rest:REST:/} interface administration and [topic](reference-terminology.md#topic) lookup.
 2. A dispatcher that handles all Pulsar [message](reference-terminology.md#message) transfers.
 
 [Brokers](reference-terminology.md#broker) can be managed via:
 
-* The [`brokers`](reference-pulsar-admin.md#brokers) command of the [`pulsar-admin`](reference-pulsar-admin.md) tool
+* The `brokers` command of the [`pulsar-admin`](https://pulsar.apache.org/tools/pulsar-admin/) tool
```

Comment:
Thank you for fixing the links one by one.  
In the future, we can add link for the first time it appears, and do not add links for other occurances.

Reply:
That would be inconvenient for users to get info.